packages/react/src/locale/hooks/useCountryAddressSchemas.ts

```diff
@@ -0,0 +1,61 @@
+import {
+  areCountriesAddressSchemasLoading,
+  areCountryAddressSchemasFetched,
+  fetchCountryAddressSchemas,
+  getCountriesAddressSchemas,
+  getCountriesAddressSchemasError,
+  getCountryAddressSchemas,
+  StoreState,
+} from '@farfetch/blackout-redux';
+import { useAction } from '../../helpers';
+import { useEffect } from 'react';
+import { useSelector } from 'react-redux';
+import type { UseCountryStatesOptions } from '../types/useCountryStates.types';
+
+export function useCountryAddressSchemas(
+  countryCode: string,
+  options: UseCountryStatesOptions = {},
```

Comment:
What do you think of defining an alias for the `UseCountryStatesOptions` type like this:

```
export type UseCountryAddressSchemasOptions = UseCountryStatesOptions;
```

In that way, I believe the user would not be confused by the naming, as he might think it was a mistake.

Reply:
Actually,  the type is created and I just put in place a wrong import. Nice catch. thanks 😉